### PR TITLE
Fix running on windows

### PR DESCRIPTION
**Pull request by Jikstra**

### CODE REVIEW

File: src/main/internal-app-schemes.ts [hunk 1]
```diff
@@ -88,23 +87,13 @@ app.once('ready', () => {
...
     const otherFolder = ALLOWED_FOLDERS.findIndex(folder => {
-      console.log(
-        process.platform,
-        file,
-        folder + '\\',
-        file.startsWith(folder + '\\')
-      )
       return process.platform === 'win32'
         ? file.startsWith(folder + '\\')
         : file.startsWith(folder + '/')
```
Comment (Simon-Laux) [inline src/main/internal-app-schemes.ts:92]: why not use `path.sep` https://nodejs.org/api/all.html#path_path_sep?

Comment (Jikstra) [reply src/main/internal-app-schemes.ts:92]: Oooh good idea :)

File: static/fix-missing-vars.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 const process = {env:{}}
-const global = {}
\ No newline at end of file
+const global = {}
+const development = 'production'
```
Comment (Jikstra) [inline static/fix-missing-vars.js:3]: I'm concerned about this @Simon-Laux what do you think? In theory this should change depending on production or development build.

Comment (Simon-Laux) [reply static/fix-missing-vars.js:3]: why did you add it here?

Comment (Jikstra) [reply static/fix-missing-vars.js]: That was what i was looking for. Didn't find it when i was scrolling through the esbuild docs :)

File: static/fix-missing-vars.js [hunk 3]
```diff
@@ -1,2 +1,3 @@
 const process = {env:{}}
-const global = {}
\ No newline at end of file
+const global = {}
+const development = process.env.NODE_ENV
```
Comment (Simon-Laux) [inline static/fix-missing-vars.js:3]: who uses this `development` - variable?

Comment (Jikstra) [reply static/fix-missing-vars.js:3]: React threw an error on windows without having this. I don't know why. Adding it and everything was good.

Comment (Simon-Laux) [reply static/fix-missing-vars.js:3]: now it is a string, but shouldn't it be a boolean?

Comment (Jikstra) [reply static/fix-missing-vars.js:3]: Nope should be a string

**Pull request merged**